src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/RuntimeBinderInternalCompilerException.cs

```diff
@@ -12,6 +12,8 @@ namespace Microsoft.CSharp.RuntimeBinder
     /// <see cref="RuntimeBinderException"/> represents a failure to bind in the sense of a usual compiler error, whereas <see cref="RuntimeBinderInternalCompilerException"/>
     /// represents a malfunctioning of the runtime binder itself.
     /// </summary>
+    [Serializable]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("Microsoft.CSharp, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a")]
```

Comment:
Is this needed? It is still in that same assembly.

Reply:
good points. I will remove it